### PR TITLE
Fix rubocop offenses in exe/jekyll

### PR DESCRIPTION
**Pull request by DirtyF**
#4885

### CODE REVIEW

File: exe/jekyll [hunk 1]
```diff
@@ -3,31 +3,35 @@ STDOUT.sync = true
 
 $LOAD_PATH.unshift File.join(File.dirname(__FILE__), *%w( .. lib ))
```
Comment (envygeeks) [inline exe/jekyll]: `()` is necessary please!

**Comment by envygeeks** [comment]
Thanks! Juts a comment.

File: exe/jekyll [hunk 2]
```diff
@@ -1,33 +1,37 @@
...
-  p.option 'destination', '-d', '--destination [DIR]', 'Destination directory (defaults to ./_site)'
-  p.option 'safe', '--safe', 'Safe mode (defaults to false)'
-  p.option 'plugins_dir', '-p', '--plugins PLUGINS_DIR1[,PLUGINS_DIR2[,...]]', Array, 'Plugins directory (defaults to ./_plugins)'
-  p.option 'layouts_dir', '--layouts DIR', String, 'Layouts directory (defaults to ./_layouts)'
-  p.option 'profile', '--profile', 'Generate a Liquid rendering profile'
+  p.description "Jekyll is a blog-aware, static site generator in Ruby"
+  p.syntax "jekyll <subcommand> [options]"
+
+  p.option "source", "-s", "--source [DIR]", "Source directory (defaults to ./)"
+  p.option "destination", "-d", "--destination [DIR]", \
```
Comment (parkr) [inline exe/jekyll]: Are the trailing `\`'s necessary here?

Comment (envygeeks) [inline exe/jekyll]: They aren't, in Ruby they are only necessary in ambiguous situations.

**Comment by parkr** [comment]
Thank you! :heart:

@jekyllbot: merge +dev

**Comment by envygeeks** [comment]
:heart:

**Pull request merged**